src/collection/collection.service.ts

```diff
@@ -977,17 +977,20 @@ export class CollectionService {
 	 *
 	 * @param collectionId Unique id of the collection that must be fetched.
 	 * @param type Type of which items should be fetched.
+	 * @param inviteToken
 	 * @returns Collection or bundle.
 	 */
 	public static async fetchCollectionOrBundleById(
```

Comment:
Maybe change the name of this function to fetchCollectionOrBundlyByIdOrInviteToken ?